ui-core/src/components/inputs/Input.tsx

```diff
@@ -86,6 +86,11 @@ const Input = React.forwardRef<HTMLInputElement, InputProps>(
         disabled={disabled}
         required={required}
         small={small}
+        statusIconPosition={
+          statusWithMessage?.tooltip === 'left' || statusWithMessage?.tooltip === 'right'
+            ? 'before-status-message'
+            : undefined
+        }
```

Comment:
This should be enough
```suggestion
        statusIconPosition={statusWithMessage?.tooltip ? 'before-status-message' : undefined}
```